plugin/evm/atomic/atomictest/shared_memories.go

```diff
@@ -0,0 +1,85 @@
+package atomictest
```

Comment:
Missing license header. (Shouldn't this fail CI?)

Reply:
Whilst we're at it, we should use something like libevm (it was a bit of a PITA to figure this out):

https://github.com/ava-labs/libevm/blob/c2e6df1b994e88af87d610420e7d52283a5d94e6/.github/workflows/lint.yml#L36-L46

Reply:
added license linter in #969